### ISSUE TITLE
getting errors

### ISSUE THREAD

**Issue by ghost** [bladerunner2020/node-red-contrib-isonline#2]
Im running node-red on a raspery pi and tried the following

[{"id":"d7df2287.4dc1a8","type":"is online","z":"96ae6f2d.e5916","name":"","url":"http://10.1.1.121/","action":"0","x":788,"y":486,"wires":[["499dcdb3.640004"]]},{"id":"d838fbb4.8040d8","type":"inject","z":"96ae6f2d.e5916","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":604,"y":485,"wires":[["d7df2287.4dc1a8"]]},{"id":"499dcdb3.640004","type":"debug","z":"96ae6f2d.e5916","name":"","active":true,"console":"false","complete":"true","x":954,"y":486,"wires":[]}]

i always get a rather strange error message no matter what i try to ser as url

msg : Object{ "_msgid": "432fb0fd.bcd05", "topic": "", "payload": false, "timestamp": 1475919053301, "online_error": "Error: connect ENOENT //10.1.1.121/", "online": false }

how do i have to declare the URL? tried without http:// tried with localhost to use a name and not an ip address, i always get the same error

**Comment by bladerunner2020** [comment]
If you enter in the browser http://10.1.1.1231/ - what to you get? 404 error?

**Comment by ghost** [comment]
accessing http://10.1.1.121 from browser or doing a curl from the node is working fine, no error

**Comment by bladerunner2020** [comment]
Could you try to specify in URL just "10.1.1.121" or "10.1.1.121:8080" or "10.1.1.121:80"?
without "http", slashes, etc.?

**Comment by ghost** [comment]
ok without "http://" and other slashes  it is working.

but "http request" node requires that msg.url: "Must start with http: or https:"
and as in "is online" "msg.url: if set overrides URL in the configuration box."  i cannot just oveeride it in the config box for the test

here what i try do achieve, without "is online" the workflow works as long the webserver responds
[{"id":"b0fc75c5.94b2c8","type":"debug","z":"96ae6f2d.e5916","name":"","active":true,"console":"false","complete":"true","x":1001,"y":669,"wires":[]},{"id":"d79c61e7.52678","type":"is online","z":"96ae6f2d.e5916","name":"","url":"","action":"1","x":676,"y":669,"wires":[["c7042e3c.3914b"]]},{"id":"af283ed7.1ca96","type":"template","z":"96ae6f2d.e5916","name":"","field":"url","fieldType":"msg","format":"handlebars","syntax":"mustache","template":"http://10.1.1.121/r?rapi=$SC+{{payload}}","x":524,"y":669,"wires":[["d79c61e7.52678"]]},{"id":"228e6546.42f2c2","type":"inject","z":"96ae6f2d.e5916","name":"","topic":"","payload":"14","payloadType":"num","repeat":"","crontab":"","once":false,"x":380,"y":669,"wires":[["af283ed7.1ca96"]]},{"id":"c7042e3c.3914b","type":"http request","z":"96ae6f2d.e5916","name":"","method":"GET","ret":"txt","url":"","tls":"","x":854,"y":668,"wires":[["b0fc75c5.94b2c8"]]}]

**Comment by javigf** [comment]
Hi team. 
I feel bad asking this , but I always get false response when url is www.google.com or google.com or ntp.com no matter what I type on url text field. But I am connecting from remote site. So my RPi must be connected to internet. Dont know what could be wrong. It is so erratic. Now when I ping from command line to the destination, sometimes shows true and some others false in 10 secs intervals. 
Thanks a lot

**Comment by bladerunner2020** [comment]
Hello!
Could you check the following flow
[{"id":"a46c7aaa.9c0108","type":"inject","z":"e2fe13c1.da894","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":102.5,"y":109,"wires":[["f3a35940.e87a18"]]},{"id":"f3a35940.e87a18","type":"is online","z":"e2fe13c1.da894","name":"","url":"","action":"0","x":269.5,"y":109,"wires":[["c4250da7.3503e8"]]},{"id":"c4250da7.3503e8","type":"debug","z":"e2fe13c1.da894","name":"","active":true,"console":"false","complete":"false","x":439.5,"y":110,"wires":[]}]

does it return false as well?
does google work ok in browser?

**Comment by javigf** [comment]
yes, now 9 out of 10 shows true. So I assume that network connectivity is not stable as expected. 
Thanks .

**Comment by akhileshappala** [comment]
HI I have been using this flow.
When I am triggering  "is online" through a API call, it is returning false once in a while even after having active internet connection.
Note : I am not polling any URL
<img width="423" alt="image" src="https://user-images.githubusercontent.com/14235798/130196930-0e81c3b7-8ab8-481f-aaac-5bb069c35ce5.png">

Thanks
